shared/Utils.js

```diff
@@ -3,5 +3,50 @@
 export default {
 	stripHtmlTags (inputString) {
 		return inputString.replace(/<\/?[^>]+(>|$)/g, "");
+	},
+
+	/**
+	 * Generate the navigation menu based on the response from the Wordpress API.
+	 * The retun menu array will:
+	 * 1. Exclude all pages whose menu_order === 0;
+	 * 2. Append "News" and "Views" menu items to the end.
+	 * @param {Array<Object>} sitePages - All menu pages sorted by menu_order.
+	 * @param {Boolean} includeHomeItem - A flag indicating if the home menu item should be included.
+	 * @return {Array<Object>} An array of object including the slug, menu title and href.
```

Comment:
For jsdoc you could setup a [typedef](https://jsdoc.app/tags-typedef.html) which would describe the object being returned. Which you'd then use in place of Object above. That's a way to provide more details on what goes into a specific type of Object.